pkg/infra/metrics/metrics.go

```diff
@@ -343,6 +355,31 @@ func init() {
 		Namespace:  ExporterName,
 	})
 
+	MRenderingRequestCompleted = newCounterStartingAtZero(prometheus.CounterOpts{
+		Name:      "rendering_request_completed_total",
+		Help:      "counter for successfully rendered panels",
+		Namespace: ExporterName,
+	})
+
+	MRenderingRequestFailed = newCounterStartingAtZero(prometheus.CounterOpts{
+		Name:      "rendering_request_failed_total",
+		Help:      "counter for panels rendered with error",
+		Namespace: ExporterName,
+	})
+
+	MRenderingExecutionTime = prometheus.NewSummary(prometheus.SummaryOpts{
+		Name:       "rendering_execution_time_milliseconds",
```

Comment:
Think this should be named `rendering_request_duration_milliseconds`, following our existing convention of `http_request_duration_milliseconds` and having labels for success/failure.